go-controller/pkg/informer/informer.go

```diff
@@ -123,7 +123,12 @@ func NewDefaultEventHandler(
 			e.enqueueDelete(obj)
 		},
 	})
-	return e
+	if err != nil {
+		klog.Warningf("Could not add Event Handler, %v", err)
```

Comment:
please change NewDefaultEventHandler to return an error, get rid of this log and propagate the error up